workers/persistedWorker.js

```diff
@@ -0,0 +1,141 @@
+/*
```

Comment:
can you remove this persistedWorker from the pull request. We want to encourage the collection/singleton worker pattern for stateful workers. Thanks!